packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconCritical from 'terra-icon/lib/icon/IconCritical';
+import IconHigh from 'terra-icon/lib/icon/IconHigh';
+import IconLow from 'terra-icon/lib/icon/IconLow';
+import IconAbnormal from 'terra-icon/lib/icon/IconAbnormal';
+import interpretationPropType from '../../proptypes/interpretationPropTypes';
+import valueQuantityPropType from '../../proptypes/valuePropTypes';
+import ResultError from '../other/_ResultError';
+import styles from './Observation.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
```

Comment:
Why isn't `_Observation.jsx` using the reusable observation proptype?

Reply:
This confused me too.  Not sure if @neilpfeiffer has a full explanation but Observations only take a subset of what's in the observation prop